radix-engine-interface/src/model/resource/non_fungible_id.rs

```diff
@@ -43,10 +40,16 @@ impl NonFungibleId {
             NonFungibleId::String(..) => NonFungibleIdType::String,
             NonFungibleId::U32(..) => NonFungibleIdType::U32,
             NonFungibleId::U64(..) => NonFungibleIdType::U64,
-            NonFungibleId::Decimal(..) => NonFungibleIdType::Decimal,
             NonFungibleId::UUID(..) => NonFungibleIdType::UUID,
         }
     }
+
+    /// Converts ordinary bytes buffer encoded as hex string
+    pub fn try_bytes_from_hex(hex_str: &str) -> Result<Self, ParseNonFungibleIdError> {
```

Comment:
I got a bit confused by this method, assuming it did SBOR decoding or something.

On reflection, I don't think we should have this method :) - people can construct more simply with `NonFungibleId::Bytes(hex::decode("abcd"))` themselves.

Reply:
Removed.